libraries/ms-helper/inc/controller_board_pins.h

```diff
@@ -0,0 +1,130 @@
+#pragma once
+// Standardized controller board ports and pins
+// Defined as a GPIO Address
+
+// CAN
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_RX \
+                                           \
+  { GPIO_PORT_A, 11 }
+
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_TX \
+                                           \
+  { GPIO_PORT_A, 12 }
+
+// USART 1
+#define CONTROLLER_BOARD_ADDR_DEBUG_USART1_TX \
+                                              \
+  { GPIO_PORT_B, 6 }
+
+#define CONTROLLER_BOARD_ADDR_DEBUG_USART1_RX \
+                                              \
+  { GPIO_PORT_B, 7 }
+
+// USART 2
+#define CONTROLLER_BOARD_ADDR_USART2_TX \
+                                        \
```

Comment:
nit: not sure why the extra blank lines between the definition and the pin, were those added by `make format`? they should be removed